<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParagraphFormatDialog</class>
 <widget class="QDialog" name="ParagraphFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupNormal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>对齐方式</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>水平</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboHAlignment">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string>左对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自动对齐</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>竖直</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboVAlignment">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string>居上</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>居下</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupIndent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>缩进</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>整体缩进</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleIndent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleFirstIndent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>首行缩进</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupSpacing">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>间距</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleAfter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>段后</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleBefor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>段前</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleLineHight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行高值</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboLineHeight">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string>单倍行距</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>多倍行距</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>固定值</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最小值</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行高策略</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupPadding">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>461</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>边距</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleTop">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleLeft">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleBottom">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleRight">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParagraphFormatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParagraphFormatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
